tsfile/src/main/java/org/apache/iotdb/tsfile/write/TsFileWriter.java

```diff
@@ -211,6 +233,22 @@ public boolean write(TSRecord record) throws IOException, WriteProcessException
     return checkMemorySizeAndMayFlushGroup();
   }
 
+  /**
+   * write a row batch
+   *
+   * @param rowBatch - series responding a data column
```

Comment:
I can hardly understand "series responding a data column".

Reply:
@jack870131  You can change this to: multiple time series of one device that share a time column

and add this as the Javadoc of RowBatch class

Reply:
Done.